targets/PythonSdk/templates/API.py.ejs

```diff
@@ -0,0 +1,34 @@
+import json
+import PlayFabErrors
+import PlayFabHTTP
+import PlayFabSettings
+
+<%- generateApiSummary("", api, "description")%>
+<% for(var i in api.calls) { var apiCall = api.calls[i];
+%><%- generateApiSummary("", apiCall, "summary")
+%><%- getDeprecationAttribute("", apiCall)
+%>def <%- apiCall.name %>(request, callback, customData = None, extraHeaders = None):
+<%- getRequestActions("    ", apiCall) %>
+    def wrappedCallback(playFabResult, error):
+<%- getResultActions("        ", apiCall, api) %>
+        callback(playFabResult, error)
+
+    PlayFabHTTP.DoPost("<%- apiCall.url %>", request, <%- getAuthParams(apiCall) %>, extraHeaders, wrappedCallback)
```

Comment:
When you fix the definiton of DoPost's last 3 parameters, you'll also need to change the last 3 here to be:
, wrappedCallback, customData, extraheaders)